### PR TITLE
import-beats: Validate ingest pipelines

### PR DESCRIPTION
**Pull request by mtojek**
This PR extends the `import-beats` script with support for validation of ingest pipelines.

### CODE REVIEW

File: dev/packages/beats/cisco/0.0.1/dataset/asa/elasticsearch/ingest-pipeline/default.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-description: "Pipeline for Cisco {< .internal_PREFIX >} logs"
+description: "Pipeline for Cisco FIX_ME logs"
```
Comment (ruflin) [inline dev/packages/beats/cisco/0.0.1/dataset/asa/elasticsearch/ingest-pipeline/default.yml:2]: NICE

File: dev/packages/beats/osquery/0.0.1/dataset/result/elasticsearch/ingest-pipeline/default.json [hunk 2]
```diff
@@ -14,14 +14,14 @@
...
-    {<if .use_namespace >}
+    
     {
       "script": {
         "lang": "painless",
         "source": "def dict = ['result': new HashMap()]; for (entry in ctx['json'].entrySet()) { dict['result'][entry.getKey()] = entry.getValue(); } ctx['osquery'] = dict; ctx.remove('json');"
       }
     },
-    {< end >}
+
```
Comment (ruflin) [inline dev/packages/beats/osquery/0.0.1/dataset/result/elasticsearch/ingest-pipeline/default.json:24]: It is valid json with these empty lines?

Comment (mtojek) [reply dev/packages/beats/osquery/0.0.1/dataset/result/elasticsearch/ingest-pipeline/default.json:24]: It passes the self-validation :)

**Comment by ruflin** [comment]
@mtojek I filed https://github.com/elastic/package-registry/issues/404 as an indirect follow up on this issue as we should also validate it during packaging.

**Comment by mtojek** [comment]
> @mtojek I filed #404 as an indirect follow up on this issue as we should also validate it during packaging.

Will try to add it in a sec.

**Comment by ruflin** [comment]
@mtojek separate PR please ;-)

**Pull request merged**